Submodule Imports/AvaloniaUI/Avalonia.DataGrid contains modified content
diff --git a/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid.Themes.Default/Avalonia.DataGrid.Themes.Default.csproj b/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid.Themes.Default/Avalonia.DataGrid.Themes.Default.csproj
index d6e449a..503bf29 100644
--- a/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid.Themes.Default/Avalonia.DataGrid.Themes.Default.csproj
+++ b/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid.Themes.Default/Avalonia.DataGrid.Themes.Default.csproj
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="0.6.2-*" />
+    <PackageReference Include="Avalonia" Version="0.8.0" />
   </ItemGroup>
   <ItemGroup>
     <Compile Update="**\*.xaml.cs">
diff --git a/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/Avalonia.DataGrid.csproj b/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/Avalonia.DataGrid.csproj
index b0189a4..b6c6f60 100644
--- a/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/Avalonia.DataGrid.csproj
+++ b/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/Avalonia.DataGrid.csproj
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="0.6.2-*" />
+    <PackageReference Include="Avalonia" Version="0.8.0" />
   </ItemGroup>
 
 </Project>
diff --git a/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/DataGridSelectedItemsCollection.cs b/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/DataGridSelectedItemsCollection.cs
index 2698b0c..ae7b875 100644
--- a/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/DataGridSelectedItemsCollection.cs
+++ b/Imports/AvaloniaUI/Avalonia.DataGrid/src/Avalonia.DataGrid/DataGridSelectedItemsCollection.cs
@@ -75,13 +75,13 @@ namespace Avalonia.Controls
             Debug.Assert(itemIndex >= 0);
 
             int slot = OwningGrid.SlotFromRowIndex(itemIndex);
-            if (_selectedSlotsTable.RangeCount == 0)
+			/*if (_selectedSlotsTable.RangeCount == 0)
             {
                 OwningGrid.SelectedItem = dataItem;
             }
-            else
-            {
-                OwningGrid.SetRowSelection(slot, true /*isSelected*/, false /*setAnchorSlot*/);
+            else*/
+			{
+				OwningGrid.SetRowSelection(slot, true /*isSelected*/, false /*setAnchorSlot*/);
             }
             return _selectedSlotsTable.IndexOf(slot);
         }
Submodule Imports/AvaloniaUI/AvaloniaEdit contains modified content
diff --git a/Imports/AvaloniaUI/AvaloniaEdit/src/AvaloniaEdit/AvaloniaEdit.csproj b/Imports/AvaloniaUI/AvaloniaEdit/src/AvaloniaEdit/AvaloniaEdit.csproj
index 886d644..c451046 100644
--- a/Imports/AvaloniaUI/AvaloniaEdit/src/AvaloniaEdit/AvaloniaEdit.csproj
+++ b/Imports/AvaloniaUI/AvaloniaEdit/src/AvaloniaEdit/AvaloniaEdit.csproj
@@ -29,7 +29,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
+    <PackageReference Include="Avalonia" Version="0.8.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
     <PackageReference Include="System.Xml.ReaderWriter" Version="4.3.1" />
   </ItemGroup>
Submodule Imports/AvaloniaUI/OxyPlot.Avalonia contains modified content
diff --git a/Imports/AvaloniaUI/OxyPlot.Avalonia/Source/OxyPlot.Avalonia/OxyPlot.Avalonia.csproj b/Imports/AvaloniaUI/OxyPlot.Avalonia/Source/OxyPlot.Avalonia/OxyPlot.Avalonia.csproj
index cdb2ac6..6b21168 100644
--- a/Imports/AvaloniaUI/OxyPlot.Avalonia/Source/OxyPlot.Avalonia/OxyPlot.Avalonia.csproj
+++ b/Imports/AvaloniaUI/OxyPlot.Avalonia/Source/OxyPlot.Avalonia/OxyPlot.Avalonia.csproj
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <ItemGroup>
     <EmbeddedResource Include="**\*.xaml;Assets\*" Exclude="bin\**;obj\**;**\*.xproj;packages\**;@(EmbeddedResource)" />
-    <PackageReference Include="Avalonia" Version="0.7.0" />
+    <PackageReference Include="Avalonia" Version="0.8.0" />
     <PackageReference Include="OxyPlot.Core" Version="2.0.0-unstable1035" />
   </ItemGroup>
 </Project>
